glean/metrics.yaml

```diff
@@ -18,6 +18,28 @@
 $schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
 
 sample:
+  android_daemon_status:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      A status report of the Android Daemon
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-3743
+    data_reviews:
+      - TODO
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - sstreich@mozilla.com
+    expires: 2023-06-01
```

Comment:
Are you sure this is enough time for validation? I made this mistake with the websocket telemetry and had to extend. Let's maybe give it a little wiggle room until 2024-01-01.

Reply:
I really would like this to be in as little as we investigate the connectionhealth->wg-go thing but good call, we can always remove stuff :)

Reply:
Alright, no strong opinions on this one :)